src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -4110,6 +4110,14 @@ DWORD ProfToEEInterfaceImpl::GetModuleFlags(Module * pModule)
 
         // Intentionally not checking for flat, since NGEN PEImages never have flat
         // layouts.
+
+#ifdef FEATURE_READYTORUN
```

Comment:
I want to support the migration of older profilers that may not be aware of R2R - throwing a new bit at them might be a breaking change. It feels like we need some way for profilers to proactively opt-in to these changes. 
Kshama Pawar on my team has been looking into this and I'd like to get her into this discussion, but I don't know if she has a GitHub account yet.